pkg/comp-functions/functions/common/billing.go

```diff
@@ -1,53 +1,124 @@
 package common
 
 import (
+	"bytes"
 	"context"
+	_ "embed"
 	"fmt"
-	v12 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 	xfnproto "github.com/crossplane/function-sdk-go/proto/v1beta1"
-	xkube "github.com/vshn/appcat/v4/apis/kubernetes/v1alpha2"
+	v1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
-	"reflect"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	controllerruntime "sigs.k8s.io/controller-runtime"
-	"strings"
+	"text/template"
 )
 
-const billingLabel = "appcat.io/billing"
+var rawExpr = "sum (kube_namespace_labels{namespace=\"{{.Namespace}}\"} * {{.Instances}})"
```

Comment:
We can simplify this. This expr will always be the value of `.Instances`

for example if .Instances = 3:
sum(1 * 3) = 3

So we can simplify this by just doing:
`vector({{ .Instances }})`

The namespace was necessary to get the `sum_over_time` working, but we moved it out of here and into the cronjobs.

Reply:
AH yes, we talked about it yesterday